events/templatetags/eventsdata.py

```diff
@@ -59,7 +59,7 @@ def participant_count(objects, category):
                 return objects.training.participant_count
             return count
         except Exception, e:
-            return 0
+            return 'N/A'
```

Comment:
Let's not handle all the exception and you can write a small comment why `N/A` is being returned.